projects/span_finder/configs/ngram/config_tok2vec.cfg

```diff
@@ -0,0 +1,140 @@
+[paths]
+train = null
+dev = null
+vectors = null
+init_tok2vec = null
+span_key = null
+suggester_model = null
+
+[system]
+gpu_allocator = "pytorch"
+seed = 0
+
+[nlp]
+lang = "en"
+pipeline = ["spancat"]
```

Comment:
Out of curiosity, why did you use a builtin `tok2vec` here instead of a separate component like you did for the `span_finder` config?

Reply:
I find the config to be more "cleaner" this way 😇